components/md.js

```diff
@@ -85,6 +85,8 @@ const md = (strings, ...values) => {
           return <LiveEdit code={literal} noInline />
         } else if (language === 'react-inline') {
           return <LiveEdit code={literal} />
+        } else if (language === 'sh') {
+          return <CodeBlock code={literal} language="bash" />
```

Comment:
So... this nail is basically because prism don't know what sh is and in order for it to work if it sh i need to set the language to bash 😢

Reply:
<img width="515" alt="screen shot 2017-12-23 at 22 03 42" src="https://user-images.githubusercontent.com/1051509/34322942-2b16d6ac-e82d-11e7-8551-eaa3bc34f17a.png">

Reply:
What's the theme?!

Reply:
VSCode ? https://marketplace.visualstudio.com/items?itemName=Equinusocio.vsc-material-theme